gui_interactions.md

```diff
@@ -0,0 +1,518 @@
+https://app.ignitionrobotics.org.
+
+In edit mode, entire models can be inserted as nested models, or broken apart
+into links to be reused by other models.
+
+In interactive mode, models are spawned into the world at runtime.
+
+### D. Manipulation
+
+Features related to changing the pose of shapes.
+
+#### D.1. Translation and rotation tool
+
+* **Modes**: Edit / Interactive
+* **Priority**: Must have
+* **References**
+    * [Ignition Citadel: Transform Control](https://ignitionrobotics.org/docs/citadel/manipulating_models#transform-control)
+* **User stories**
+    * User wants to test the robot on a different place in the world, without
+      restarting simulation.
+* **Status**: ✅ Available since Acropolis.
+
+#### D.2. Align
+
+* **Modes**: Edit / Interactive
+* **Priority**: Should have
+* **References**
+    * [Ignition Citadel: Align Tool](https://ignitionrobotics.org/docs/citadel/manipulating_models#align-tool)
+* **User stories**
+* **Status**: ✅ Available since Citadel.
+
+#### D.3. Joint manipulation
+
+* **Modes**: Edit / Interactive
+* **Priority**: Could have
+* **References**
+* **User stories**
+* **Status**: ⌛
+
+Manual joint manipulation through its range of motion.
+
+### E. Workflow
+
+Features related to the user's workflow.
+
+#### E.1. Property inspector
+
+* **Modes**: Edit / Interactive
+* **Priority**: Must have
+* **References**
+    * [Gazebo tutorial](http://gazebosim.org/tutorials?tut=model_editor#Edityourmodel)
+    * [Ignition Citadel: Component Inspector](https://ignitionrobotics.org/docs/citadel/manipulating_models#component-inspector)
+* **User stories**
+    * User has an SDF model and wants to change the mass of a link.
+    * User has an SDF light and wants to change its color.
+    * User has an SDF model and wants to change the position of a joint.
+* **Status**: ✅ Read-only inspector for several component types, as well as pose editing,
+available from Blueprint.
+
+More component types need to be made writable, according to the current
+simulation mode.
+
+#### E.2. 3D Visualizations
+
+* **Modes**: Edit / Interactive
+* **Priority**: Should have
+* **References**
+    * [Issue: Visualize inertia](https://github.com/ignitionrobotics/ign-gazebo/issues/111)
+    * [Issue: Visualize center of mass](https://github.com/ignitionrobotics/ign-gazebo/issues/110)
+    * [Issue: Visualize joints](https://github.com/ignitionrobotics/ign-gazebo/issues/106)
+    * [Issue: Visualize collision shapes](https://github.com/ignitionrobotics/ign-gazebo/issues/105)
+* **User stories**
+    * User wants to debug a model spawned into simulation.
+* **Status**: ⌛
+
+Visualization of entities other than visuals, such as collisions, joints,
+sensors, lights, etc.
+
+Similar to Gazebo-classic, these can be reachable from a context menu.
+Visibility options could also be added to the component inspector. Rendering
+visuals will need to be implemented.
+
+#### E.3. Undo / redo
```

Comment:
How many times can a user undo? Most of the software I've used let the user set the number of undos that can be performed since it has a big effect on memory.

Reply:
Thanks, Cole. Added a note in a77b9ed.